src/commands/convert/index.ts

```diff
@@ -8,7 +28,9 @@ import {Command, Flags} from '@oclif/core'
 import Zap2HDF from './zap2hdf'
 
 function getInputFilename(): string {
-  const inputFileIndex = process.argv.findIndex(param => param.toLowerCase() === '-i' || param.toLowerCase() === '--input')
+  const inputFileIndex = process.argv.findIndex(
```

Comment:
there seem to be a bunch of changes in this file that are linting changes but i don't think we changed anything about the linter so this might be stuff that changed based off of your editor settings.  please revert these formatting changes unless the linter is complaining about them.